### ISSUE TITLE
Are there any plans to make ruffle.rs multilingual?

### ISSUE THREAD

**Issue by creeper-0910** [ruffle-rs/ruffle-rs.github.io#110]
Are there any plans to make https://ruffle.rs multilingual?

**Comment by n0samu** [comment]
I think that would be a really good idea!

**Comment by Herschel** [comment]
I agree! Could use a bit of discussion on Discord to determine the best course forward. I could see having a region drop-down similar to [https://www.rust-lang.org/].

**Comment by Dinnerbone** [comment]
Ideally we use the current crowdin project for sourcing translations: https://crowdin.com/project/ruffle/
The rewrite to nextjs should make this more viable to do.
